### ISSUE TITLE
Abandoned

### ISSUE THREAD

**Issue by maiorano84** [vuejs-tips/vue-the-mask#137]
Please mark this project as abandoned. No changes or PRs have been addressed in over 2 years.

**Comment by shrpne** [comment]
Looks like @kaskar2008 is maintaining the fork https://github.com/KazanExpress/ke-the-mask
Maybe @neves can give him access to contribute?

**Comment by shrpne** [comment]
I have looked at the other issues and found many more forks #135 #112 #90, so I guess some of them can maintain this repo. We only need @neves to give them rights.

**Comment by ErickPetru** [comment]
@neves você ainda tá vivo? Entrega o repositório para alguém que ainda pretende contribuir.

**Comment by mateusavila** [comment]
@neves teu projeto é foda, tu fez bastante coisa legal pra comunidade. Mas libera pra geral manter o projeto. Estamos tentando corrigir uns bugs, e quanto mais olhos olham um problema, menor ele fica.

**Comment by DinsmoreDesign** [comment]
Agreed, there are a handful of issues with this that need to be fixed, but trying to sort through forks isn't ideal. I'd just switch to something like Cleave, but their implementation is even more buggy and the API is far less elegant.

**Comment by RonaldJerez** [comment]
@DinsmoreDesign I have reached to Neves directly via email and have not gotten many responses, i also see his website vue-js-tips hasn't really been updated. So not sure if something has happened to him but it's safe to say that non of these messages are getting to him and thus no one will be getting maintenance rights for this repo.  

For my part I renamed my fork and maintaining it as a separate project now.  The codebase has diverged but the API is still very similar to vue-the-mask. Give a try and see if that works better for you.

https://ronaldjerez.github.io/vue-input-facade/

**Comment by DinsmoreDesign** [comment]
> @DinsmoreDesign I have reached to Neves directly via email and have not gotten many responses, i also see his website vue-js-tips hasn't really been updated. So not sure if something has happened to him but it's safe to say that non of these messages are getting to him and thus no one will be getting maintenance rights for this repo.
> 
> For my part I renamed my fork and maintaining it as a separate project now. The codebase has diverged but the API is still very similar to vue-the-mask. Give a try and see if that works better for you.
> 
> https://ronaldjerez.github.io/vue-input-facade/

Very cool! I'll definitely check it out!
